sonar-plugin-api/src/main/java/org/sonar/api/batch/bootstrap/ProjectDefinition.java

```diff
@@ -44,7 +44,7 @@
 
   public static final String TESTS_PROPERTY = "sonar.tests";
 
-  public static final String BUILD_DIR_PROPERTY = "sonar.buildDir";
+  public static final String ANALYZE_PARENT_MODULES_PROPERTY = "sonar.analyzeParentModules";
```

Comment:
Same: since ProjectDefinition is part of the API, I would not add public stuff in it.